src/components/Admin/UserGroup/UserGroupPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react'
+import { useUserGroupSWR, useUserGroupRelationsSWR } from '~/stores/admin';
+import { useAclEnabled } from '~/stores/context';
+
+import UserGroupTable from '~/client/js/components/Admin/UserGroup/UserGroupTable';
+import UserGroupCreateForm from '~/client/js/components/Admin/UserGroup/UserGroupCreateForm';
+import UserGroupDeleteModal from '~/client/js/components/Admin/UserGroup/UserGroupDeleteModal';
+
+import { toastSuccess, toastError } from '~/client/js/util/apiNotification';
+import { apiv3Get, apiv3Delete } from '~/utils/apiv3-client';
+import { UserGroup, UserGroupRelation } from '~/interfaces/user';
+
+const UserGroupPage = (): JSX.Element => {
```

Comment:
```suggestion
export const UserGroupPage = (): JSX.Element => {
```

FC にするのであれば named export の方がいいと思います

Reply:
元の jsx を使わなくなったのであれば削除お願いします 🙏